app/src/main/java/com/github/se/studybuddies/ui/screens/ChatScreen.kt

```diff
@@ -79,7 +80,7 @@ fun ChatScreen(viewModel: MessageViewModel, navigationActions: NavigationActions
     }
   }
 
-  OptionsDialog(viewModel, selectedMessage, showOptionsDialog, showEditDialog)
+  OptionsDialog(viewModel, selectedMessage, showOptionsDialog, showEditDialog, navigationActions)
   EditDialog(viewModel, selectedMessage, showEditDialog)
 
   Column(
```

Comment:
Also try to match the general esthetic of the Figma and UI structure. Meaning try to use MainScreenScaffold for the chat screen. The 4 main screens should have a drawer menu and a bottom navigation bar.